source/slang/glsl.meta.slang

```diff
@@ -2368,24 +2592,60 @@ public vector<T,4> textureLod(__TextureImpl<
 [require(glsl_hlsl_spirv, texture_shadowlod)]
 public float textureLod(sampler2DShadow sampler, vec3 p, float lod)
 {
-    // TODO: Need to apply lod
-    return sampler.SampleCmp(p.xy, p.z);
+    __requireComputeDerivative();
```

Comment:
OpImageSampleDrefExplicitLod does not need `requireComputeDerivative`.

It is only needed for implicitLod functions (the derivative is used to figure out a Lod).